packages/owc-dev-server/README.md

```diff
@@ -33,26 +33,41 @@ Please note that the owc-dev-server *only resolves bare module specifiers*, and
 :::
 
 
-### Usage
+## Usage
+If you just want to try owc-dev-server once, you can [run it through `npx`](#execute-with-npx). If you decide to keep using owc-dev-server in your project, you should install and add it to your package's `devDependencies`:
 ```bash
 npm i -D owc-dev-server
+# or
+yarn add --dev owc-dev-server
 ```
 
-#### Execute
-```
+Note: The owc-dev-server package is *named after the command* it provides. That's why it doesn't follow the naming pattern for the other open-wc packages and does not use a `@open-wc/` scope prefix.
+
+### Execute as script
+For convenience (i.e. to always use the same [command line options](#command-line-options)) you should add an entry to your `scripts`:
+```javascript
 // package.json
 "scripts": {
   "start": "owc-dev-server"
 }
+```
 
-// bash
+This allows you to execute owc-dev-server like this:
+```bash
+# run package.json script 'start'
 npm run start
+# or
+yarn run start
+```
 
-// or via npx
+### Execute with npx
+You can also start owc-dev-server through `npx`. This allows you to easily try out different [command line options](#command-line-options). Starting owc-dev-server through `npx` will even work if you did not previously install it ([see above](#usage)).
```

Comment:
can you use npx without installing a module? will that install it on the fly?

Reply:
jup that's the idea behind npx 🤗 

however we had some path problems with `owc-dev-server` when it was used via npx... not 100% if we have them all fixed 🙈

Reply:
Nope, it's not fixed yet for owc-dev-server. Just verified with a fresh app (using `polymer init --name polymer-3-starter-kit`). Sorry, I assumed wrongly without trying it first. 🙈
I will adjust the parts *usage* and *execute with npx* accordingly.

Reply:
FYI: I created #366 so I can reference it in the docs.